scripts/modules/helpers.py

```diff
@@ -82,6 +83,74 @@ def load_images(urls, cache_dir):
 DEFAULT_HEALTHCHECK_RETRIES = 12
 
 
+def _print_done(service_name):
+    # 22 chars to ...
+    done = '\033[32mdone'
+    num_spaces = 21 - len(service_name)
+    s = ' '
+    dots = '...'
+    for _ in range(num_spaces):
+        s = s + ' '
+    print("{service}{spaces}{dots} {done}".format(service=service_name, spaces=s, dots=dots, done=done))
+
+
+def try_to_set_slowlog(password):
+    # This is a bit tricky to follow. What we're doing here is forking a "manager"
+    # process that will occasionally attempt to configure the slow log. After it
+    # detects that the slowlog has successfully been configured, it declares victory
+    # and terminates itself.
+    manager_process = multiprocessing.Process(target=_set_slowlog_json, args=(password,))
+    manager_process.start()
+
+
+def _set_slowlog_json(password):
+    this_try = 0
+    tries = 30
+    while this_try <= tries:
+        this_try += 1
+        time.sleep(3)
+        completed_process = subprocess.run([
+            "curl",
```

Comment:
Instead of forking a process could you do this in python - perhaps using the [Retry](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.retry.Retry) package?

Example:
```py
response = http.request(
  'PUT', 
  'localhost:9200/_settings?pretty', 
  retries=Retry(30)
  headers={'Content-Type': 'application/json'},
  body=json.dumps({...})
)
```

Reply:
IIRC, the reason we fell back to a shell invocation here is that we already have cURL installed and didn't want to introduce an additional Python dependency.

Reply:
afaict `urllib` is already available: https://github.com/elastic/apm-integration-testing/blob/9221ec337e6e158f20cc0997a0804275d650d22d/requirements.txt#L38

But no biggie - I was just wondering when I saw this.

Reply:
Ah, you're right! Thanks for pointing this out. :)